vividus-plugin-mobile-app/src/main/resources/properties/profile/mobile_app/profile.properties

```diff
@@ -4,3 +4,4 @@ mobile-app=true
 mobile-application.session-scope=story
 mobile-application.swipe.stabilization-duration=PT1S
 mobile-application.swipe.limit=10
+mobile-application.hide-keyboard-using-native-call=true
```

Comment:
it is not an attribute of mobile application

Reply:
ios.hide-keyboard-using-native-call - are you ok with it?